src/commercetools/testing/abstract.py

```diff
@@ -118,6 +118,16 @@ def delete_by_key(self, key):
             if obj["key"] == key:
                 return self.objects.pop(obj_id)
 
+    def delete_by_container_key(self, container, key):
```

Comment:
Maybe we should name it `delete_by_container_and_key` to avoid users misreading it as deleting a bunch of objects by container key. But this depends on whether this naming convention is used elsewhere in a similar way.